src/panels/config/logs/system-log-card.ts

```diff
@@ -228,12 +260,16 @@ export class SystemLogCard extends LitElement {
     fireEvent(this, "switch-log-view");
   }
 
+  private async _prepareDownloadUrl() {
+    const downloadUrl = getErrorLogDownloadUrl;
+    const { path } = await getSignedPath(this.hass, downloadUrl);
```

Comment:
How long is this url valid for? And how long are we going to show it to the user? There might be a real possibility the url is expired before the user clicks it.

Reply:
Oh, I never thought of that. I'll check